plugins/samanage/unit_test/parameters/assign_incident.json.resp

```diff
@@ -0,0 +1,35 @@
+{
+	"parameters": [
+		[
+			"11111",
+			"user@example.com",
+			{
+				"incident": {
+					"id": 10000,
+					"number": "1000",
+					"name": "Incident Example Name",
+					"description": "Example incident description",
+					"state": "New",
+					"assignee": {
+                    "group_id": 1,
+                    "is_user": "true",
+                    "id": 1,
+                    "name": "Example User",
+                    "email": "user@example.com",
+                    "avatar": {
+                        "type": "initials",
+                        "color": "#fd4165",
+                        "initials": "UE"
+                    }
+                },
+					"site": {
+						"id": 1,
+						"name": "Austin TX, USA",
+						"location": "AUS"
+					}
+				}
+
+			}
+		]
+	]
+}
```

Comment:
Could you please reformat the code here, and add new line at the end? 😄

Reply:
Done 👍